bentoml/_internal/models/model.py

```diff
@@ -102,11 +108,16 @@ def from_fs(cls, item_fs: FS) -> "Model":
         try:
             with item_fs.open(MODEL_YAML_FILENAME, "r") as model_yaml:
                 info = ModelInfo.from_yaml_file(model_yaml)
+            if item_fs.isfile(CUSTOM_OBJECTS_FILENAME):
+                with item_fs.open(CUSTOM_OBJECTS_FILENAME, "rb") as cofile:
+                    custom_objects = cloudpickle.load(cofile)
```

Comment:
we can add a `custom_objects` property method in Model class, where it reads the custom objects from `fs` lazily upon access?

Reply:
I pushed a new commit for this change